ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/policies/.content.xml

```diff
@@ -58,6 +60,15 @@
                             </assetToComponentMapping>
                         </cq:authoring>
                     </policy_xfm>
+                    <policy_600116016309972
```

Comment:
@DeepaliAgrwl @rakeshkumar-verma ; why do we have a policy-xfm here ?

Reply:
@rishabh-behl  it is the default policy provided. I didn't removed the existing policies just created the new policies for hcleomm